callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
in what ways do the hasher implementations in this file differ from the password hashers in Django (besides not using the term "password")?